libs/clients/passports/src/lib/passports.config.ts

```diff
@@ -0,0 +1,28 @@
+import { defineConfig } from '@island.is/nest/config'
+import { NationalRegistryScope } from '@island.is/auth/scopes'
+import * as z from 'zod'
+
+const schema = z.object({
+  xRoadServicePath: z.string(),
+  fetch: z.object({
+    timeout: z.number().int(),
+    scope: z.array(z.string()),
+  }),
+})
+
+export const PassportsClientConfig = defineConfig<z.infer<typeof schema>>({
+  name: 'PassportsClient',
+  schema,
+  load(env) {
+    return {
+      xRoadServicePath: env.required(
+        'XROAD_PASSPORT_LICENSE_PATH',
+        'IS-DEV/GOV/10001/SKRA-Protected/Forskraning-V1',
+      ),
+      fetch: {
+        timeout: 10000,
+        scope: [NationalRegistryScope.passport],
```

Comment:
What's the status of this change on their side? Will they perform it in all environments soon, or do we need to include the individual scope for compatibility?

Reply:
I did not really get a timeline on that, only the confirmation that it could be changed to /passport.

Perhaps it would be best to change it back and have both until I get the confirmation that it will be ok to remove the /individuals scope. So we can be safe in all environments.

I'll do that.